Formula/minio.rb

```diff
@@ -31,9 +31,9 @@ def install
         proj = "github.com/minio/minio/"
 
         system "go", "build", "-o", buildpath/"minio", "-ldflags", <<-EOS.undent
-            -X #{proj}/cmd.Version=#{version}
-            -X #{proj}/cmd.ReleaseTag=#{release}
-            -X #{proj}/cmd.CommitID=#{commit}
+            -X #{proj}cmd.Version=#{version}
```

Comment:
Actually this looks weird. could you please just remove the trailing slash from `proj`'s definition and leave these intact?